src/modules/accounting.py

```diff
@@ -0,0 +1,251 @@
+
+        (
+            last_requests_id_to_finalize,
+            finalization_shares_amount,
+        ) = self._get_requests_finalization_report(block_hash)
+        logger.info({
+            'msg': 'Get withdrawals queue stats.',
+            'last_requests_id_to_finalize': last_requests_id_to_finalize,
+            'finalization_shares_amount': finalization_shares_amount,
+        })
+        if last_requests_id_to_finalize:
+            LAST_FINALIZED_WITHDRAWAL_REQUEST.set(last_requests_id_to_finalize[-1])
+
+        transaction = contracts.oracle.functions.handleCommitteeMemberReport((
+            int(slot / self.slots_per_epoch),
+            beacon_validators_count,
+            beacon_validators_balance,
+            stacking_module_ids,
+            no_operators,
+            exited_validators,
+            wc_balance,
+            wei_to_reserve,
+            last_requests_id_to_finalize,
+            finalization_shares_amount,
+        ))
+
+        return transaction
+
+    def _get_beacon_validators_stats(self, slot: SlotNumber, block_hash: HexBytes) -> Tuple[int, int]:
+        # Should return balance in gwei!
+        lido_validators = get_lido_validators(self._w3, block_hash, self._beacon_chain_client, slot)
+        return len(lido_validators), sum(int(validator['validator']['balance']) for validator in lido_validators)
+
+    def _get_new_exited_validators(
+            self,
+            slot: SlotNumber,
+            block_hash: HexBytes,
+    ) -> Tuple[List[str], List[int], List[int]]:
+        validators = self._get_exited_validators(slot, block_hash)
+
+        exited_validators = defaultdict(int)
+
+        for validator in validators:
+            exited_validators[(validator['key']['module_id'], validator['key']['operator_index'])] += 1
+
+        stacking_module_id = []
+        node_operator_id = []
+        exited_validators_number = []
+
+        operators = get_lido_node_operators(self._w3, block_hash)
+        for operator in operators:
+            exited_validators_prev = operator['stoppedValidators']
+
+            exited_validators_current = exited_validators[(operator['module_id'], operator['index'])]
+
+            if exited_validators_prev != exited_validators_current:
+                stacking_module_id.append(operator['module_id'])
+                node_operator_id.append(operator['index'])
+                exited_validators_number.append(exited_validators_current)
+
+        return (
+            stacking_module_id,
+            node_operator_id,
+            exited_validators_number,
+        )
+
+    def _get_exited_validators(self, slot: SlotNumber, block_hash: HexBytes) -> List[MergedLidoValidator]:
+        lido_validators = get_lido_validators(self._w3, block_hash, self._beacon_chain_client, slot)
+
+        withdrawable_validators = filter(lambda validator: int(validator['validator']['validator']['exit_epoch']) <= int(slot / self.slots_per_epoch), lido_validators)
```

Comment:
We cannot rely on `exit_epoch` since a validator may still have eth on the balance
1) There may be validators with non-0x01 type WC
2) There is a time gap between `exit` and `withdrawable`

https://kb.beaconcha.in/ethereum-2.0-depositing#exited
https://notes.ethereum.org/7CFxjwMgQSWOHIxLgJP2Bw#4311-Initiate-exit

An example of a slashed and exited validator with eth:
http://unstable.mainnet.beacon-api.nimbus.team/eth/v1/beacon/states/head/validators/3280

```json
{
  "execution_optimistic": false,
  "data": {
    "index": "3280",
    "balance": "34118878082",
    "status": "withdrawal_possible",
    "validator": {
      "pubkey": "0xb3622c1180e86dcfe82e3e2a9d76a27614c002ba0dc9c058f8d6d5c75b95b5e834a30d9b84269ec8171419009c4cc5c3",
      "withdrawal_credentials": "0x00c4bc4e7b2e72415bf73c6af63cc13e4ebc98d1c87167d2a539771e38047381",
      "effective_balance": "32000000000",
      "slashed": true,
      "activation_eligibility_epoch": "0",
      "activation_epoch": "0",
      "exit_epoch": "148724",
      "withdrawable_epoch": "156911"
    }
  }
}
```